WalletWasabi.Fluent/Models/Windows/WindowsCapture.cs

```diff
@@ -0,0 +1,644 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using Avalonia;
+using Avalonia.Media.Imaging;
+using Avalonia.Platform;
+
+namespace WalletWasabi.Fluent.Models.Windows;
+#pragma warning disable IDE0019 // Use pattern matching
+#pragma warning disable IDE1006 // Naming Styles
+
+public class WindowsCapture
```

Comment:
Try to run this code (Open QR code window) for a few hours and see if the memory is growing. I would like to make sure there is no memory leak in this library.